src/data/samurai-dreams.json

```diff
@@ -0,0 +1,25 @@
+{
+  "slug": "samurai-dreams",
+  "title": "Samurai Dreams",
+  "featuredTitle": "Samurai Dreams",
+  "description": "A tale of two samurai.",
+  "quote": "You won’t understand anything. Everything is set up in the first two.",
```

Comment:
This is not technically a quote from Samurai Dreams (none of the footage shown has any dialogue) but spoken in Videodrome about Samurai Dreams.